lambdas/build.sh

```diff
@@ -1,11 +1,7 @@
 #!/bin/bash
 
-#Installing node 6.10
-NODE_VERSION="6.10"
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
-nvm install ${NODE_VERSION}
-nvm use ${NODE_VERSION}
```

Comment:
Do we need to keep the `nvm install` and `nvm use` instructions? Just not specify the node version?

Reply:
yes it should still run those commands, and use the `.nvmrc` for the version

Reply:
Done